### ISSUE TITLE
Navigator temporary stops working if it was triggered inside a bottom sheet after closing the opened page

### ISSUE THREAD

**Issue by Hitsu91** [jonataslaw/getx#764]
**Description**
I encountered this bug when if use the navigator [Get.to(newPage), that will not close the bottom sheet, with Get.off works fine because the bottom sheet will be closed] inside the bottom sheet. It works fine the first time it's tapped, but, if I close the opened page, the navigator will not works as expected; the button itself works perfectly, i tried to print random values inside the function and it works. This applies to every other buttons inside the bottom sheet that use the navigator too.
Only after closing the bottom sheet and reopened it, the navigator return to works properly.

**Reproduction code**

```dart

import 'package:flutter/material.dart';
import 'package:get/get.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return GetMaterialApp(
      title: 'Navigator Bug',
      home: MyHomePage(),
    );
  }
}

class MyHomePage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('HOME PAGE'),
      ),
      body: Center(
        child: RaisedButton(
          onPressed: () {
            Get.bottomSheet(
              MyBottomSheet(),
              backgroundColor: Get.theme.canvasColor,
            );
          },
          child: Text('Open bottom sheet'),
        ),
      ),
    );
  }
}

class MyBottomSheet extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Container(
      child: Column(
        mainAxisAlignment: MainAxisAlignment.center,
        children: [
          RaisedButton(
            onPressed: () {
              Get.to(NewPage());
            },
            child: Text('Get.to'),
          ),
          RaisedButton(
            onPressed: () {
              Get.to(NewPage());
            },
            child: Text('An other Get.to'),
          ),
          RaisedButton(
            onPressed: () {
              Get.off(NewPage());
            },
            child: Text('Get.off'),
          ),
        ],
      ),
    );
  }
}

class NewPage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(),
      body: Center(
        child: Text('Hello'),
      ),
    );
  }
}

```

**To Reproduce**
1. Open the bottom sheet
2. Click one of the button that use the Get.to function
3. Close the newly opened page
4. Try to reuse the navigator with one of the other buttons
5. Close the bottom sheet
6. The navigator will works as aspected again

**Expected behavior**
Navigator shouldn't stop working after close the opened page.

**Flutter Version:**
 1.22.2

**Getx Version:**
3.12.1
3.15

**Describe on which device you found the bug:**
- Xiaomi Mi9
- iOS simulator iphone 12 PRO

**Comment by jonataslaw** [comment]
Duplicate of https://github.com/jonataslaw/getx/issues/514
